projects/canopy/src/lib/header/primary-navigation/primary-navigation.component.scss

```diff
@@ -0,0 +1,163 @@
+@import '../../../styles/mixins';
+
+$primary-nav-border-bottom-height: 0.25rem;
+
+.lg-primary-nav-wrapper {
+  flex: 1;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  overflow: visible;
+}
+
+.lg-primary-nav {
+  bottom: 0;
+  left: 0;
+  padding: var(--space-sm);
+  position: fixed;
+  right: 0;
+  top: calc(var(--header-height) + var(--space-xxxs));
+  display: none;
+  overflow: visible;
+  background: var(--color-white);
+  flex-grow: 1;
+
+  &--active {
+    display: block;
+  }
+
+  @include lg-breakpoint(md) {
+    bottom: auto;
+  }
+
+  @include lg-breakpoint(lg) {
+    display: flex;
+    justify-content: flex-end;
+    position: static;
+    top: 0;
+    padding: 0;
+  }
+}
+
+.lg-primary-nav-list {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: column;
+  align-items: flex-start;
+  list-style-type: none;
+  margin: 0;
+  padding: var(--space-xxxs) 0 0 0;
+  width: 100%;
+
+  @include lg-breakpoint('lg') {
```

Comment:
Not a blocker 😅 can you either write `@include lg-breakpoint('lg')` or `@include lg-breakpoint(lg)` for all of them?